pkg/apis/kubedirector.bluedata.io/v1alpha1/kubedirectorapp_types.go

```diff
@@ -101,26 +101,26 @@ type ServiceEndpoint struct {
 type NodeRole struct {
 	ID           string           `json:"id"`
 	Cardinality  string           `json:"cardinality"`
-	ImageRepoTag *string          `json:"image_repo_tag,omitempty"`
-	SetupPackage SetupPackage     `json:"config_package,omitempty"`
-	PersistDirs  *[]string        `json:"persist_dirs"`
-	MinResources *v1.ResourceList `json:"min_resources"`
+	ImageRepoTag *string          `json:"imageRepoTag,omitempty"`
+	SetupPackage SetupPackage     `json:"configPackage,omitempty"`
+	PersistDirs  *[]string        `json:"persistDirs"`
+	MinResources *v1.ResourceList `json:"minResources"`
 }
 
 // NodeGroupConfig identifies a set of roles, and the services on those roles.
 // The top-level config indicates which roles and services will always be
 // active. Implementation of "config choices" will introduce other conditional
 // configs.
 type NodeGroupConfig struct {
-	RoleServices   []RoleService     `json:"role_services"`
-	SelectedRoles  []string          `json:"selected_roles"`
-	ConfigMetadata map[string]string `json:"config_meta"`
+	RoleServices   []RoleService     `json:"roleServices"`
```

Comment:
Do you think it would be possible to add this as a check in the Travis job? Would that be a good idea?